### ISSUE TITLE
When install the py_ecc  by pip, it raises error: 'error: README.md: No such file or directory'

### ISSUE THREAD

**Issue by poshboytl** [ethereum/py_ecc#1]
Platform: MacOS 10.12.5

OutPut: 

```
~/work/pyethereum(develop) $ pip install py_ecc
Collecting py_ecc
/Users/terrytai/.pyenv/versions/2.7.6/lib/python2.7/site-packages/pip/_vendor/requests/packages/urllib3/util/ssl_.py:318: SNIMissingWarning: An HTTPS request has been made, but the SNI (Subject Name Indication) extension to TLS is not available on this platform. This may cause the server to present an incorrect TLS certificate, which can cause validation failures. You can upgrade to a newer version of Python to solve this. For more information, see https://urllib3.readthedocs.io/en/latest/security.html#snimissingwarning.
  SNIMissingWarning
/Users/terrytai/.pyenv/versions/2.7.6/lib/python2.7/site-packages/pip/_vendor/requests/packages/urllib3/util/ssl_.py:122: InsecurePlatformWarning: A true SSLContext object is not available. This prevents urllib3 from configuring SSL appropriately and may cause certain SSL connections to fail. You can upgrade to a newer version of Python to solve this. For more information, see https://urllib3.readthedocs.io/en/latest/security.html#insecureplatformwarning.
  InsecurePlatformWarning
  Using cached py_ecc-1.1.0.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/gd/2jkp088j3d96slmx_5gllmc40000gn/T/pip-build-vmYHRm/py-ecc/setup.py", line 6, in <module>
        with open('README.md') as f:
    IOError: [Errno 2] No such file or directory: 'README.md'

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /private/var/folders/gd/2jkp088j3d96slmx_5gllmc40000gn/T/pip-build-vmYHRm/py-ecc/
```

I found the setup.py try to open README.md and LICENSE, but the two files are not in the package: https://pypi.python.org/pypi/py_ecc

**Comment by nizar-jamal** [comment]
This error is still present. So, I did a:
pip download py_ecc
untarred the tgz
touch README.md and LICENSE files in the py_ecc-1.1.2 directory
and ran the setup.py install

**Comment by mikeshultz** [comment]
1.1.1 is unaffected, by the way.

    pip install py_ecc==1.1.1

**Comment by palkeo** [comment]
I have the same issue, but with the LICENSE file.

**Comment by pipermerriam** [comment]
Issue should be fixed in `1.4.1`
